Sources/Core/Presenter/Channel/ChannelPresenter+Events.swift

```diff
@@ -183,7 +183,7 @@ extension ChannelPresenter {
     
     private func shouldMessageEventBeHandled(_ message: Message) -> Bool {
         guard let parentMessage = parentMessage else {
-            return message.parentId == nil
+            return message.parentId == nil || message.showReplyInChannel
```

Comment:
Why don't we use `isThread` here too?

Reply:
This is different. It's about to check if a message has a parent message id.

Reply:
How is it different? Message events for thread messages should not be handled, hence we can use `isThread`

Reply:
I now see that `isThread` is defined in `ChannelPresenter` and not in `Message`. Then it's fine

Reply:
`isThread` is a property of `ChannelPresenter`
To avoid `message.parentId == nil` we can add `iisReply` for `Message`.